recipes/qt_binder/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2" %}
+
+package:
+  name: qt_binder
+  version: {{ version }}
+
+source:
+  fn: qt_binder-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/q/qt_binder/qt_binder-{{ version }}.tar.gz
+  sha256: 392a105a6a1734e1f67e17b88ee72225be594dfc0000dc3d53e254a1b8a532e5
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - libpng >=1.6.22  # current qt 4.8.7 3 from defaults requires libpng 1.6.22
```

Comment:
@grlee77 currently your pinning here is no different from what we are using everywhere else (`libpng >=1.6.21,<1.7`) because we don't have a `1.7` version anywhere. So this is OK and consistent for now, but as we update `libpng` this will get inconsistent. So it is better safe than sorry. Can you change that to `libpng >=1.6.21,<1.7` so we can merge this one?